packages/teams-js/test/public/webStorage.spec.ts

```diff
@@ -65,9 +66,14 @@ describe('webStorage', () => {
               } else {
                 // not supported for any client type with invalid version
                 it(`webStorage.isWebStorageClearedOnUserLogOut should not allow call for context ${frameContext}, hostClientType ${clientType} and version ${version}`, async () => {
-                  await framedPlatformMock.initializeWithContext(frameContext, clientType);
-                  framedPlatformMock.setRuntimeConfig(generateBackCompatRuntimeConfig(version));
-                  expect(webStorage.isWebStorageClearedOnUserLogOut()).not.toBeTruthy();
+                  if (
+                    GlobalVars.hostClientType === HostClientType.web ||
+                    HostClientType.desktop === GlobalVars.hostClientType
```

Comment:
can be changed to  GlobalVars.hostClientType === HostClientType.web || GlobalVars.hostClientType  === HostClientType.desktop for better readability.

Reply:
Resolved in latest commit